pkgs/top-level/python-packages.nix

```diff
@@ -7533,6 +7533,8 @@ in {
 
   rxv     = callPackage ../development/python-modules/rxv     { };
 
+  frameioclient = callPackage ../development/python-modules/frameioclient { };
+
```

Comment:
please attempt to sort this, will cause merge conflicts if everyone just appends to the end

Reply:
I've placed it in a section that has a bunch of packages starting with 'f'. I can't seem to piece together an actual ordering so i hope this will suffice.

Reply:
that's fine :)